aerogear-android-push/src/main/java/org/jboss/aerogear/android/unifiedpush/gcm/UnifiedPushConfig.java

```diff
@@ -245,7 +245,7 @@ public UnifiedPushConfig setAlias(String alias) {
      * 
      */
     public UnifiedPushConfig setCategories(List<String> categories) {
-        this.categories = new ArrayList<String>(categories);
+        this.categories = categories;
```

Comment:
Weird, I just reverted this file and reran the automated tests and it works.
Is there a problem when it is running in the demo or something?